docs/source/utils.md

```diff
@@ -72,6 +72,16 @@ INFO [osp.core.ontology.parser]: Loaded 205 ontology triples in total
 INFO [osp.core.ontology.installation]: Uninstallation successful
 ```
 
+### Ontology installation folder
+
+The installed ontologies are stored in the directory `~/.osp-ontologies` by 
+default. On Windows, `~` usually refers to the path 
+`C:\Users\username`.
```

Comment:
I just had a flashback of some people once asking on the internet why something is failing because they thought that literally their username would be between `<>`. But yes this is better, and hopefully our users will not think this way.